prs/PizzaStore.ts

```diff
@@ -0,0 +1,40 @@
+class PizzaStore {
+    public createPizza(style: string, type: string): Pizza | null {
+        let pizza: Pizza = undefined;
+
+        if (style === "NY") {
```

Comment:
This if statement is very large - consider using some additional design patterns here.  Take a look at [Factory Method](https://en.wikipedia.org/wiki/Factory_method_pattern) and [Factory Pattern](https://en.wikipedia.org/wiki/Factory_(object-oriented_programming))

Reply:
Or [Builder pattern](https://en.wikipedia.org/wiki/Builder_pattern)